easybuild/easyconfigs/v/VSCode/VSCode-1.85.1-cli-x64.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'Tarball'
+
+name = 'VSCode'
+version = '1.85.1'
+versionsuffix = '-cli-x64'
+
+homepage = 'https://code.visualstudio.com/'
+description = '''
+    Visual Studio Code is a lightweight but powerful source code editor 
+    which runs on your desktop and is available for Windows, macOS and 
+    Linux. It comes with built-in support for JavaScript, TypeScript and 
+    Node.js and has a rich ecosystem of extensions for other languages 
+    and runtimes (such as C++, C#, Java, Python, PHP, Go, .NET). Begin 
+    your journey with VS Code with these introductory videos.
+'''
+
+toolchain = {'name': 'system', 'version': 'system'}
+
+source_urls = ['https://update.code.visualstudio.com/%(version)s/cli-alpine-x64/stable#']
+sources = ['vscode_cli_alpine_x64_cli.tar.gz']
```

@laraPPr We should rename this source tarball on download, since it doesn't include the version:

```suggestion
sources = [{
    'download_filename': 'vscode_cli_alpine_x64_cli.tar.gz',
    'filename': 'vscode-%(version)s%(versionsuffix)s.tar.gz',
}]